src/components/NetworkStatusService.js

```diff
@@ -60,13 +58,8 @@ goog.require('ga_waitcursor_service');
     var count = 0;
     var promise;
     this.$get = function($document, $rootScope, $timeout, $window,
-        gaBrowserSniffer, gaGlobalOptions) {
-      if (!gaBrowserSniffer.mobile) {
-        return {
-          offline: false,
-          check: function() {}
-        };
-      }
+        gaGlobalOptions) {
+
```

Comment:
There was a circular dependency injection with gaUrlUtils using $http and since the iPad, for example, uses the desktop version I think it makes sense to remove this conditionnal use of the geNetworkService.